bundles/binding/org.openhab.binding.powermax/src/main/java/org/openhab/binding/powermax/PowerMaxBindingConfig.java

```diff
@@ -0,0 +1,71 @@
+/**
+ * Copyright (c) 2010-2016, openHAB.org and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.powermax;
+
+import org.openhab.binding.powermax.internal.PowerMaxSelectorType;
+import org.openhab.core.binding.BindingConfig;
+import org.openhab.core.items.Item;
+
+/**
+ * Binding Configuration class. Represents a binding configuration in the items
+ * file to a PowerMax Alarm system
+ *
+ * @author lolodomo
+ * @since 1.9.0
+ */
+
+public class PowerMaxBindingConfig implements BindingConfig {
+
+    private PowerMaxSelectorType selectorType;
+    private String selectorParam;
+    private Item item;
+
+    public PowerMaxBindingConfig(PowerMaxSelectorType selectorType, String selectorParam, Item item) {
+        this.selectorType = selectorType;
+        this.selectorParam = selectorParam;
+        this.item = item;
+    }
+
+    /**
+     * @return the selector type
+     */
+    public PowerMaxSelectorType getSelectorType() {
+        return selectorType;
+    }
+
+    /**
+     * @return the selector parameter, or null if no parameter
+     */
+    public String getSelectorParam() {
+        return selectorParam;
+    }
+
+    /**
+     * @return the item
+     */
+    public Item getItem() {
+        return item;
+    }
+
```

Comment:
Checking my code, I only call getItem in my code to know the kind of the item (StringItem or SwitchItem or ...).
If you think that is better, I will store here the "kind of item" (class name ?) rather than the item itself ?